addons/website_slides/controllers/mail.py

```diff
@@ -64,6 +64,7 @@ def mail_update_message(self, res_model, res_id, message, message_id, attachment
         message.sudo().write({
             'body': message_body,
             'attachment_ids': [(4, aid) for aid in attachment_ids],
+            'date': fields.Datetime.now(),
```

Comment:
why this change ? should it be the date of the last modification ? (I can not find this in the task pad)

Reply:
Hi, this was suggested by mog during testing. I will ask lna about it